setup.py

```diff
@@ -1,20 +1,52 @@
-import setuptools
+import re
+from codecs import open
+from os import path
+
+from setuptools import find_packages, setup
+
+PACKAGE_NAME = "discord_slash"
+HERE = path.abspath(path.dirname(__file__))
 
 with open("README.md", "r", encoding="UTF-8") as f:
-    long_description = f.read()
+    README = f.read()
+with open(path.join(HERE, PACKAGE_NAME, "const.py"), encoding="utf-8") as fp:
+    VERSION = re.search('__version__ = "([^"]+)"', fp.read()).group(1)
+
+extras = {
+    "lint": ["black", "flake8", "isort"],
+    "readthedocs": ["sphinx", "sphinx-rtd-theme"],
+}
+extras["lint"] += extras["readthedocs"]
+extras["dev"] = extras["lint"] + extras["readthedocs"]
 
-setuptools.setup(
+setup(
     name="discord-py-slash-command",
-    version="1.2.2",
+    version=VERSION,
     author="eunwoo1104",
     author_email="sions04@naver.com",
     description="A simple discord slash command handler for discord.py.",
-    long_description=long_description,
+    extras_require=extras,
+    install_requires=["discord.py", "aiohttp"],
+    license="MIT License",
+    long_description=README,
     long_description_content_type="text/markdown",
     url="https://github.com/eunwoo1104/discord-py-slash-command",
-    packages=setuptools.find_packages(),
-    python_requires='>=3.6',
+    packages=find_packages(),
+    python_requires=">=3.6",
     classifiers=[
-        "Programming Language :: Python :: 3"
-    ]
+        "Development Status :: 4 - Beta",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: MIT License",
+        "Natural Language :: English",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
```

Comment:
```suggestion
```
d.py currently doesn't support 3.9

Reply:
Python 3.x is compatible with Python 3.9.

Reply:
There's a reason they don't support it, i can't exactly remember but i think it was something to do with it not building well on 3.9? Not sure if it was resolved

Reply:
I see, that doesn't make much sense though. By design, each version of Python 3.x must be compatible with each prior version preceding it.

Reply:
This reminds me of https://xkcd.com/1172/ :smile:
It seems like it was because dependencies didn't support 3.9, they've said d.py works on 3.9 but don't explicitly include it in their setup.py
Not sure what to do honestly

Reply:
Wasn't this issue with 3.9.x resolved a while ago? it was a dep issue that afaik is now fixed. Also, subjectively, all of my bots are running on 3.9.x without issue